test/eqDeep.js

```diff
@@ -0,0 +1,104 @@
+/* jshint typed: true */
+
+var assert = require('assert');
+
+var R = require('..');
+
+describe('eqDeep', function() {
+    var a = [];
+    var b = a;
+    it('tests for deep equality of its operands', function() {
+        assert.strictEqual(R.eqDeep(100, 100), true);
+        assert.strictEqual(R.eqDeep(100, '100'), false);
+        assert.strictEqual(R.eqDeep([], []), true);
+        assert.strictEqual(R.eqDeep(a, b), true);
+    });
+
+    it('handles objects', function() {
+        assert.strictEqual(R.eqDeep({}, {}), true);
+        assert.strictEqual(R.eqDeep({a:1, b:2}, {a:1, b:2}), true);
+        assert.strictEqual(R.eqDeep({a:2, b:3}, {b:3, a:2}), true);
+        assert.strictEqual(R.eqDeep({a:2, b:3}, {a:3, b:3}), false);
+        assert.strictEqual(R.eqDeep({a:2, b:3, c:1}, {a:2, b:3}), false);
+    });
+
+    var supportsSticky = false;
+    try { RegExp('', 'y'); supportsSticky = true; } catch (e) {}
+
+    var supportsUnicode = false;
+    try { RegExp('', 'u'); supportsUnicode = true; } catch (e) {}
+
+    it('handles regex', function() {
+        assert.strictEqual(R.eqDeep(/\s/, /\s/), true);
+        assert.strictEqual(R.eqDeep(/\s/, /\d/), false);
+        assert.strictEqual(R.eqDeep(/a/gi, /a/ig), true);
+        assert.strictEqual(R.eqDeep(/a/mgi, /a/img), true);
+        assert.strictEqual(R.eqDeep(/a/gi, /a/i), false);
+
+        if (supportsSticky) {
+            // assert.strictEqual(R.eqDeep(/\s/y, /\s/y), true);
+            // assert.strictEqual(R.eqDeep(/a/mygi, /a/imgy), true);
+        }
+
+        if (supportsUnicode) {
+            // assert.strictEqual(R.eqDeep(/\s/u, /\s/u), true);
+            // assert.strictEqual(R.eqDeep(/a/mugi, /a/imgu), true);
+        }
```

Comment:
The idea was to leave them as reminders to attach them when we are properly able to run them: https://github.com/ramda/ramda/pull/718#commitcomment-9422717

Do you have serious reservations?  I would love it if we could get 0.9.0 out the door this weekend.  And I think this is the only major item left.